packages/fxa-customs-server/lib/config/config.js

```diff
@@ -475,6 +475,64 @@ module.exports = function (fs, path, url, convict) {
           },
         },
       },
+      recoveryPhoneCreateCodeRules: {
+        actions: {
+          doc:
+            'Array of actions that this rule should be applied to. A user should only be able to initiate ' +
+            'a recovery phone setup 9 times a day.',
+          default: ['recoveryPhoneCreate'],
+          format: Array,
+        },
+        limits: {
+          max: {
+            doc: 'max actions during `period` that can occur before rate limit is applied',
+            format: 'nat',
+            default: 9,
+            env: 'RECOVERY_PHONE_SETUP_CODE_RULE_MAX',
+          },
+          periodMs: {
+            doc: 'period needed before rate limit is reset',
+            format: 'duration',
+            default: '1 day',
+            env: 'RECOVERY_PHONE_SETUP_CODE_RULE_PERIOD_MS',
```

Comment:
Question: Should this be `_MS` if the duration is '1 day'? Is any duration format fine?

Reply:
I think any duration is fine, convict will convert it to milliseconds.